src/mscorlib/src/System/Diagnostics/Stacktrace.cs

```diff
@@ -125,7 +125,7 @@ internal void InitializeSourceInfo(int iSkip, bool fNeedFileInfo, Exception exce
                 if (s_symbolsMethodInfo == null)
                 {
                     s_symbolsType = Type.GetType(
-                        "System.Diagnostics.StackTraceSymbols, System.Diagnostics.StackTrace, Version=4.0.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a",
+                        "System.Diagnostics.StackTraceSymbols, System.Diagnostics.StackTrace, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a",
```

Comment:
Typically we want to probe for the lowest version and let the loader pick a newer version if that's what is available.  That's what will happen here, right @gkhanna79?